examples/browser/webpack.config.js

```diff
@@ -28,5 +34,15 @@ module.exports = require("../../config/webpack/webpack.config")(__dirname, {
         port: process.env.PORT
     }, plugins: [
         new webpack.HotModuleReplacementPlugin()
-    ]
-});
\ No newline at end of file
+    ],
+    resolve: {
+	// These shims are needed for bunyan
+	alias: {
+            'dtrace-provider': path.resolve(__dirname, './webpack_empty.js'),
+            fs: path.resolve(__dirname, './webpack_empty.js'),
```

Comment:
OK Seems like mocking wiht a simple : 'empty' is way nicer too I'll check if that works